app/models/user.rb

```diff
@@ -1,6 +1,8 @@
 class User < ActiveRecord::Base
   include AndelaValidator
   include ActionView::Helpers::DateHelper
+  include NewNotification
+
```

Comment:
This is the way the notification system knows when a new user is created on Zhishi. It seems to also be useful when/if we want to be sending some kind of thank you for joining zhishi mail, but largely it's how the NotificationSystem knows to create or map new users.